run_operations.py

```diff
@@ -332,31 +352,32 @@ def post_processing(files_to_move):
 
 			# Write that stored date was overwritten
 			overwrite_stored_date()
-			
+
 			# Submit job via process_sentinel and store output
 			logger.info("%s: STARTING PROCESS SENTINEL JOB AT: %s (newest date: %s)\n", dataset, psen_time, most_recent)
 			files_to_move = run_process_sentinel()
-			
+
 			all_output_files += files_to_move;
-			
+
 		else:
 			logger.info("%s: NO NEW DATA on %s (most recent: %s)\n", dataset, psen_time, stored_date)
 
 		logger.removeHandler(info_handler)
 		logger.removeHandler(error_handler)
-		
+
 		# Perform post processing on all of the output and error files produced by processSentinel
 		while len(all_output_files) != 0:
 			for i, file in enumerate(all_output_files):
 				if os.path.exists(file) and os.path.isfile(file):
-					files_to_move = [all_output_files[i], all_output_files[i+1]]
+					files_to_move = [all_output_files[i], all_output_files[i + 1]]
 					post_processing(files_to_move)
-				
-					all_output_files[:] = [f for fi in files_to_move if fi not in file and fi not in all_output_files[i+1]]
-				
+
+					all_output_files[:] = [f for fi in files_to_move if
```

Comment:
What is this doing? Hard to understand, so a refactor might be in order.

Reply:
Something definitely to be done now is to change the variable name `file` to something else as `file` is a builtin

Reply:
Let's move this to a separate diff. I don't want to mess with this change, so I'll wait until this PR lands to `master`